gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -301,6 +274,105 @@ def run(self) -> GtsfmData:
             process_graph_generator.is_image_correspondence = True
         process_graph_generator.save_graph()
 
+        if self.parsed_args.run_single_trial:
+            # TODO(johnwlambert): Check if these are None, otherwise don't override.
+            COMBINATIONS = [(self.parsed_args.num_matched, self.parsed_args.max_frame_lookahead)]
+        else:
+            # Run multiple trials at different (num_matched, max_frame_lookahead) settings.
+            COMBINATIONS = [
+                (0, 5),
+                (0, 10),
+                (5, 0),
+                (5, 5),
+                (5, 10),
+                (10, 0),
+                (10, 5),
+                (10, 10),
```

Comment:
potentially useful to make the max values here configurable, not needed in this PR though